packages/jest-core/src/collectHandles.ts

```diff
@@ -42,6 +42,10 @@ const alwaysActive = () => true;
 // @ts-expect-error: doesn't exist in v10 typings
 const hasWeakRef = typeof WeakRef === 'function';
 
+function asycSleep(milliseconds: number) {
+  return new Promise(resolve => setTimeout(resolve, milliseconds));
```

Comment:
can we use `util.promisify`? i.e. `const asyncSleep = promisify(setTimeout)`

Reply:
I didn’t do this because I assumed it wouldn’t work (since the callback is first instead of last). But I just checked and I guess it does! Will change.